glean-core/android-native/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="mozilla.telemetry.glean_native" >
+    <application>
```

Comment:
This shouldn't be exposing [debug activity](https://searchfox.org/glean/rev/0fc3f5092b3dd1015abb7bc725a36eea50b7ec32/glean-core/android/src/main/AndroidManifest.xml#4) as well, right? I don't think so because that's still something `android` needs to do

Reply:
It shouldn't (so this code here is correct). 
Consumers will depend on the glean package, which does expose the debug activity.